.github/workflows/dev-deploy-worker.yml

```
@@ -0,0 +1,160 @@
+name: Deploy DEV Worker
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - next
+      - main
+    paths:
+      - 'package.json'
+      - 'yarn.lock'
+      - 'apps/api/**'
+      - 'libs/dal/**'
+      - 'libs/shared/**'
+env:
+  TF_WORKSPACE: novu-dev
+
+jobs:
+  test_worker:
+    strategy:
+      matrix:
+        name: ['novu/worker', 'novu/worker-ee']
+    uses: ./.github/workflows/reusable-worker-e2e.yml
+    with:
+      ee: ${{ contains (matrix.name,'ee') }}
+
+  deploy_dev_worker:
+    deploy_dev_webhook:
+      if: "!contains(github.event.head_commit.message, 'ci skip')"
+      # The type of runner that the job will run on
+      runs-on: ubuntu-latest
+      needs: test_webhook
+      timeout-minutes: 80
+      environment: Development
+      permissions:
+        contents: read
+        packages: write
+        deployments: write
+      steps:
+        - uses: actions/checkout@v3
+        - uses: ./.github/actions/setup-project
+
+        - name: Build, tag, and push image to Amazon ECR
+          id: build-image
+          env:
+            REGISTRY_OWNER: novuhq
+            DOCKER_NAME: novu/worker
```

```suggestion
            DOCKER_NAME: novu/worker-ee
```

For dev we only build EE